contrib/cmus-autoupdate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+usage() {
+    printf "%s <music directory>
+    Automatically add/remove files to/from the Library
+    when files in the given directory are changed." $0
+    exit
+}
+
+[ -z "$1" ] && usage
+
+which inotifywait >/dev/null || { printf "This script requires inotify-tools to be installed."; exit; }
+
+# NB. if your files are on removable storage, -e unmount may be useful here.
+inotifywait --csv --monitor --recursive -e create -e delete "$1" | \
+while IFS=, read -r path event file; do
+	file="$(printf "%s\n" "$file" | sed -e 's/^"//;s/"$//')"
```

Comment:
To avoid this splitting logic and to simplify the parsing, using the `--format` argument with `%e|%w%f` and using `while IFS='|' read -r event path` should be simpler and handle all special characters properly.